web/lens/src/styles.css

```
@@ -3670,32 +3696,43 @@
        longer reading as columns on an axis at all. Width encodes nothing here;
        height does. */
     --lens-waterfall-bar-width: min(50%, 96px);
+    /* The working height of the bars — see the chart's rows, where it becomes a
+       floor the plot may grow past. Named so the printed bridge can drop it to
+       nothing without restating the row template. */
+    --lens-waterfall-plot-h: 288px;
     scrollbar-width: thin;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)